### ISSUE TITLE
GMRT packages no longer installed after updating to 2024.800.0.605

### ISSUE THREAD

**Issue by KormexGit** [YoYoGames/GameMaker-Bugs#7179]
### Description
I had all packages installed in .602, and after installing .605, they're all gone and need to be installed again. Interestingly, if I re-install .602 again without installing the packages in .605, they re-appear. 
If I do install all the packages in .605 and then reinstall .605, they are still there. So it seems the two versions are using separate sets of packages.

### Steps To Reproduce
Install IDE. 2024.800.0.602. Install all of the GMRT packages. Install IDE 2024.800.0.605. The packages will be gone, unless you previously installed them with .605.

### Which version of GameMaker are you reporting this issue for?
IDE v2024.800.0.605 Runtime v2024.800.0.628

### Which operating system(s) are you seeing the problem on?
Windows 10.0.22631.0

[6cad3886-9b71-406f-ab1c-709f15222e96](https://api.gamemaker.io/api/github/downloads/6cad3886-9b71-406f-ab1c-709f15222e96)

**Comment by YYDan** [comment]
Yes, the location has changed in recent Betas, as the previous default location used in Preferences was set prior to the changes to move projects out of My Documents so GameMaker avoids "the OneDrive problems".

The newer IDE is correct/this is not a bug.

**Comment by YYDan** [comment]
Tidied up now: https://github.com/YoYoGames/GameMaker-Bugs/issues/7182
